xml/System.Net/ServicePoint.xml

```diff
@@ -317,9 +317,9 @@
  ]]></format>
         </remarks>
         <exception cref="T:System.ArgumentOutOfRangeException">The value specified for a set operation is a negative number less than -1.</exception>
-        <altmember cref="P:System.Net.HttpWebRequest.KeepAlive" />
-        <altmember cref="P:System.Net.FtpWebRequest.KeepAlive" />
-        <altmember cref="P:System.Net.HttpListenerRequest.KeepAlive" />
+        <altmember cref="P:System.Net.HttpWebRequest.KeepAlive?displayProperty=nameWithType" />
```

Comment:
These aren't displayed, so there's no reason to add ?displayProperty=nameWithType

Reply:
It is actually part of the docs - See Also section: https://docs.microsoft.com/en-us/dotnet/api/system.net.servicepoint.connectionleasetimeout?view=netcore-2.0#see-also

Reply:
Thanks for pointing that out, @karelz. It's a new addition, and I hadn't noticed it yet.

Reply:
I don't think there's any way to configure the link display with the current schema since this portion is pure ECMAXML so the displayProperty is not accepted. When we move to Markdown, we'll be able to change the display of these links.